lib/pure/strutils.nim

```diff
@@ -2660,41 +2660,73 @@ func findNormalized(x: string, inArray: openArray[string]): int =
               # security hole...
   return -1
 
-func invalidFormatString() {.noinline.} =
-  raise newException(ValueError, "invalid format string")
+func invalidFormatString(explain: string) {.noinline.} =
+  ## Raise value erorr for invalid interpolation format string
+  raise newException(ValueError, "invalid format string - " & explain)
```

Comment:
Not a blocker but I prefer more specific error kinds, make more precise exception handling possible without shenanigans.